e2e-tests/tls-issue-cert-manager/run

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+test_dir=$(realpath $(dirname $0))
+. ${test_dir}/../functions
+
+checkTLSFiles() {
+  for var in '$@'; do
+    if [ -z "$var" ]; then
+      echo 'cant find required tls file'
+      exit 1
+    fi
+  done
+}
+
+checkTLSSecrets() {
+  crt={$(getTLSSecret some-name-ssl 'ca.crt'):-''}
+  tlsCert={$(getTLSSecret some-name-ssl 'tls.crt'):-''}
+  tlsKey={$(getTLSSecret some-name-ssl 'tls.key'):-''}
+
+  checkTLSFiles "$crt" "$tlsCert" "$tlsKey"
+}
```

Comment:
if we. use functions in three places - please move them to functions

Reply:
768fdf1356a0b7f1be2c6057ca45dd7045809929